src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -50,6 +50,24 @@ public ResourceElement CreateSearchBundle(SearchResult result)
                     {
                         Mode = r.SearchEntryMode == SearchEntryMode.Match ? Bundle.SearchEntryMode.Match : Bundle.SearchEntryMode.Include,
                     },
+
+                    // Response is an optional section which we add if we want to alert the client
+                    // that the response was truncated.
+                    Response = r.IsPartial == false ? null : new Bundle.ResponseComponent
```

Comment:
Does this mean we might include this more than once? If so perhaps the message should be formatted to indicate which result has truncated includes

Reply:
yes we add this PER resource. this is according to spec:
<img width="1243" alt="Screen Shot 2020-07-16 at 8 02 25" src="https://user-images.githubusercontent.com/13205761/87629048-e3142400-c73a-11ea-804d-4d577c82e177.png">

https://www.hl7.org/fhir/STU3/bundle.html#resource
https://www.hl7.org/fhir/bundle.html#resource

Don't think that we should indicate which resource it refers to, since the hierarchy already implies it

Reply:
I don't thisnk that's what we should be doing. We should have one entry at the end of the bundle with the warning. Like when you hit vonk with an unkown parameter: https://vonk.fire.ly/Patient?_format=json&missingparameter=x

Reply:
The difference is that unknown parameter is true for all resources and truncation per entity is higher resolution.